src/jdk.compiler/share/classes/com/sun/tools/javac/tree/DocTreeMaker.java

```diff
@@ -480,8 +480,13 @@ public DCUses newUsesTree(ReferenceTree name, List<? extends DocTree> descriptio
 
     @Override @DefinedBy(Api.COMPILER_TREE)
     public DCValue newValueTree(ReferenceTree ref) {
```

Comment:
Revert the implementation of this method.

Reply:
That's not necessary, because we're in the private implementation world here.  The important change you're suggesting is to the methods in the `DocTreeFactory` interface.